packages/@aws-cdk/aws-codebuild/lib/project-logs.ts

```diff
@@ -0,0 +1,78 @@
+import * as logs from '@aws-cdk/aws-logs';
+import * as s3 from '@aws-cdk/aws-s3';
+
+/**
+ * Information about logs built to an S3 bucket for a build project.
+ */
+export interface S3LogsOptions {
+  /**
+   * Encrypt the S3 build log output
+   *
+   * @default true
+   */
+  readonly encrypted?: boolean;
+
+  /**
+   * The S3 Bucket to send logs to
+   */
+  readonly bucket: s3.IBucket;
+
+  /**
+   * The path prefix for S3 logs
+   *
+   * @default - no prefix
+   */
+  readonly prefix?: string;
+
+  /**
+   * The current status of the logs in Amazon CloudWatch Logs for a build project
+   *
+   * @default true
+   */
+  readonly enabled?: boolean;
+}
+
+/**
+ * Information about logs built to a Cloudwatch Log Group for a build project.
+ */
+export interface CloudwatchLogsOptions {
+  /**
+   * The Log Group to send logs to
+   *
+   * @default - no log group specified
+   */
+  readonly logGroup?: logs.ILogGroup;
+
+  /**
+   * The prefix of the stream name of the Amazon CloudWatch Logs
+   *
+   * @default - no prefix
+   */
+  readonly prefix?: string;
+
+  /**
+   * The current status of the logs in Amazon CloudWatch Logs for a build project
+   *
+   * @default true
+   */
+  readonly enabled?: boolean;
+}
+
+/**
+ * Information about logs for the build project. A project can create logs in Amazon CloudWatch Logs, an S3 bucket, or both.
+ */
+export interface LogsOptions {
```

Comment:
Let's rename this to `LoggingOptions`, to make it consistent with the property name.

Let's also rename `S3LogsOptions` to `S3LoggingOptions`, and `CloudwatchLogsOptions` to `CloudWatchLoggingOptions`.